src/Http/Routing/src/Matching/DfaMatcherBuilder.cs

```diff
@@ -808,5 +942,46 @@ private static bool TryGetRequiredValue(RoutePattern routePattern, RoutePatternP
 
             return !RouteValueEqualityComparer.Default.Equals(value, string.Empty);
         }
+
+        // TODO: Convert to record struct when available
+        [DebuggerDisplay("{" + nameof(GetDebuggerDisplay) + "(),nq}")]
+        private struct DfaBuilderWorkerWorkItem
```

Comment:
Record struct now available?

Reply:
I'll check for this and try out removing the #pragma